src/main/java/net/stickycode/plugin/frontmatter/FrontmatterRules.java

```diff
@@ -11,7 +11,8 @@
 
   public void add(FrontmatterRule... rules) {
     for (FrontmatterRule rule : rules) {
-      this.rules.add(rule);
+      if (!this.rules.contains(rule))
+        this.rules.add(rule);
```

Comment:
Do you perhaps want to fail on this instead of silently ignore it?

Reply:
good idea